pages/common/mongorestore.md

```diff
@@ -0,0 +1,19 @@
+# mongorestore
+
+> Utility to import a collection or database from a binary dump into a MongoDB instance.
+
+- Import database from folder containing a bson data dump:
+
+`mongorestore -h {{database_host}} -d {{database_name}} {{path/to/folder}}`
+
+- Import database with user authentication from a folder containing bson data dump:
```

Comment:
Maintaining parity with other sql-ish(see `mongo`, `myql`) pages, its best to prompt user for passwords rather than type it on the command prompt. The man page says to prompt for password, one should just drop the `--password` flag.

How does this sound -  "Import a bson data dump from a folder to a given database on a given host with a given username; user will be prompted for password" ? And we can just drop the `-p {{password}}` part.